sys-cluster/crmsh/crmsh-4.4.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{9,10} )
```

Comment:
Python 3.11 wont work?

Reply:
did not try this, as said above, not experienced enough ;)